webhook_server_container/libs/github_api.py

```diff
@@ -759,6 +759,42 @@ def close_issue_for_merged_or_closed_pr(self, hook_action):
                 issue.edit(state="closed")
                 break
 
+    @ignore_exceptions(logger=FLASK_APP.logger)
+    def delete_remote_tag_for_merged_or_closed_pr(self, hook_action):
+        pr_tag = f"pr-{self.pull_request.number}"
+        # run regctl as a container:
+        base_regctl_command = (
+            "podman run -i --rm --net host  -v regctl-conf:/home/appuser/.regctl/ ghcr.io/regclient/regctl:latest"
+        )
+        registry_info = self.container_repository.split("/")
+        registry_url = "" if len(registry_info) < 2 else registry_info[0]
```

Comment:
sorry, should be 3

quay.io/openshift-cnv/cnv-tests-github << with domain
myakove/my-image >> without domain